libs/ui/src/lib/line-chart/line-chart.component.ts

```diff
@@ -123,7 +123,7 @@ export class LineChartComponent implements AfterViewInit, OnChanges, OnDestroy {
     const marketPrices = [];
 
     this.historicalDataItems?.forEach((historicalDataItem, index) => {
-      benchmarkPrices.push(this.benchmarkDataItems?.[index]?.value);
+      benchmarkPrices.push(this.benchmarkDataItems?.[index]?.value || null);
```

Comment:
I wouldn't set null values in the line-chart component, as this component is not only used for the average unit price, but also for the net performance on the home page where zero shouldn't be hidden.

Reply:
Good point